melodic/distribution.yaml

```diff
@@ -4069,6 +4069,22 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2018.3.25-0
     status: maintained
+  librealsense2:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/IntelRealSense/librealsense2-release.git
```

Comment:
There's no melodic releases in the repository you cannot add a release section here without running bloom for melodic.